docs/general/plugin-ca.md

```diff
@@ -0,0 +1,194 @@
+
+### Cause of the traffic disruptions
+We can simply add a new `cacerts` secret with our certificates but what does it mean for service to service mTLS traffic? Consider `sleep` and `httpbin` demo applications. Both are sharing the same root of trust which is the Istio CA generated self-signed root certificate:
+```bash
+$ istioctl proxy-config secret deployment/sleep -n sleep -o json | jq -r '.dynamicActiveSecrets[1].secret.validationContext.trustedCa.inlineBytes' | base64 --decode | openssl x509 -text -noout
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            b0:3c:9a:d8:3f:ab:c8:a7:12:23:a8:65:d6:5e:76:c8
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: O=cluster.local
+        Validity
+            Not Before: Jun 19 12:39:46 2025 GMT
+            Not After : Jun 17 12:39:46 2035 GMT
+        Subject: O=cluster.local
+        ...
+$ istioctl proxy-config secret deployment/httpbin -n httpbin -o json | jq -r '.dynamicActiveSecrets[1].secret.validationContext.trustedCa.inlineBytes' | base64 --decode | openssl x509 -text -noout
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            b0:3c:9a:d8:3f:ab:c8:a7:12:23:a8:65:d6:5e:76:c8
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: O=cluster.local
+        Validity
+            Not Before: Jun 19 12:39:46 2025 GMT
+            Not After : Jun 17 12:39:46 2035 GMT
+        Subject: O=cluster.local
+        ...
+```
+
+Now, when you create your new `cacerts` secret with an intermediate certificate signed by a different root certificate, you need to restart all workloads to pick up the change. There will be a time window where one of the workloads is already using a new certificate signed by the new intermediate certificate while the other workload is still using the old one, so the mTLS communication breaks, causing traffic disruption.
+
+### Avoiding traffic disruptions
+To achieve the no-downtime update of the certificates, it's necessary to ensure that all workloads at any given time are trusting certificates signed by either the old root or the new intermediate (which is signed by the new root). This can be achieved by enabling Istio's multi-root support.
+
+1. Enable the multi root support:
+
+    Prepare`istio-patch.yaml`:
+    ```yaml
+    apiVersion: sailoperator.io/v1
+    kind: Istio
+    spec:
+      values:
+        pilot:
+          env:
+            ISTIO_MULTIROOT_MESH: "true"
```

Comment:
Do you need this setting if we are combining both roots into a single PEM?

Reply:
`If enabled, mesh will support certificates signed by more than one trustAnchor for ISTIO_MUTUAL mTLS`
Not sure if it makes sense to reference Istio upstream doc for clarity here, I can add it.